docusaurus.config.js

```diff
@@ -91,7 +91,7 @@ module.exports = {
         redirects: [
           { to: '/spicedb-dedicated/overview', from: '/products/dedicated' },
           { to: '/spicedb-self-hosted/overview', from: '/products/enterprise' },
-		  { to: '/spicedb-self-hosted/overview', from: '/spicedb-enterprise/overview' },
+		      { to: '/spicedb-self-hosted/overview', from: '/spicedb-enterprise/overview' },
```

Comment:
this indentation seems like it was an accident

Reply:
This was to fix a tabs vs spaces instance